pennylane/qnn/cost.py

```diff
@@ -0,0 +1,115 @@
+"""
+This submodule contains frequently used loss and cost functions.
+"""
+# pylint: disable=too-many-arguments, too-few-public-methods
```

Comment:
Is the `too-few-public-methods` disable necessary?

Reply:
Right, now that there are some methods it's no longer useful. Removed.